### PR TITLE
Add support for rtx

### PR DESCRIPTION
**Pull request by dnicolson**
This adds support for [rtx](https://github.com/jdx/rtx).

### All submissions

* [x] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
* [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change

### Adding/updating Application X Support

* [x] This PR is only for one application
* [x] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
* [x] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
* [x] Syncing does not break the application
* [x] Syncing does not compete with any syncing functionality internal to the application
* [x] The configuration syncs the minimal set of data
* [x] No file specific to the local workstation is synced
* [x] No sensitive data is synced

### Improving the Mackup codebase

* [x] My submission passes the [tests](https://github.com/lra/mackup/tree/master/tests)
* [x] I have linted the code locally prior to submission
* [ ] I have written new tests as applicable
* [ ] I have added an explanation of what the changes do

### CODE REVIEW

File: mackup/applications/rtx.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[application]
+name = rtx
+
+[configuration_files]
+.config/rtx
```
Comment (lra) [inline mackup/applications/rtx.cfg:5]: This should be moved in a `xdg_configuration_files` section like explained at https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory

File: mackup/applications/rtx.cfg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+.rtx.toml
+.tool-versions
```
Comment (lra) [inline mackup/applications/rtx.cfg:7]: This file and the ones below seem totally generic, are those really owned by rtx? Why does it have both a `.config` folder and lots of files in the user's home?

Comment (dnicolson) [reply mackup/applications/rtx.cfg:7]: These files relate to [asdf](https://github.com/lra/mackup/blob/64d6687/mackup/applications/asdf.cfg) and rtx, which is an [asdf rust clone](https://github.com/lra/mackup/blob/64d6687/mackup/applications/asdf.cfg).

File: mackup/applications/rtx.cfg (hunk 1 above)
Comment (dnicolson) [reply mackup/applications/rtx.cfg:5]: I will make this change to `asdf` as well.

File: mackup/applications/rtx.cfg (hunk 2 above)
Comment (lra) [reply mackup/applications/rtx.cfg:9]: Makes sense, thanks.

**Review by lra — APPROVED** [review]

**Pull request merged**